Resources/public/css/theme/alloyeditor/general.css

```diff
@@ -4,9 +4,13 @@
  */
 
 [class*="ae-icon-"], [class*=" ae-icon-"] {
-    font-size: 18px;
+    font-size: 1.067em;
```

Comment:
It is a strange-looking value. What's the actual size (in pixels)?

Reply:
Actually, this is miscalculation from my side:  this size should be 18px = 1.2em. Ref: https://github.com/ezsystems/PlatformUIBundle/pull/957#pullrequestreview-105251196